packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/TableRows/index.js

```diff
@@ -20,12 +20,14 @@ const TableRows = ({ headers, rows, onOpenModal }) => {
     }
 
     if (type === 'action') {
+      const [actionName, ...actionIdentifiers] = value.split('.');
+
       return formatMessage(
         {
-          id: `Settings.permissions.auditLogs.${value}`,
-          defaultMessage: getDefaultMessage(value),
+          id: `Settings.permissions.auditLogs.actions.${actionName}`,
+          defaultMessage: getDefaultMessage(actionName),
         },
-        { model }
+        { model, action: actionIdentifiers.join('') }
```

Comment:
@gu-stav This is maybe specific to our the audit logs use case but the `select` doesn't accept a format like `auth.success` so I have to make it a string it likes `authsuccess`. Also, I think the logic I'm doing here works currently but might break down depending on new events we potentially add in the future.